lib/ActivityStreams.js

```diff
@@ -209,6 +220,48 @@ ActivityStreams.prototype = {
   },
 
   /**
+   * Returns an object of functions with results cached
+   */
+  _get_memoized(cache) {
```

Comment:
I actually intended to call it memoize after the term "memoization": https://en.wikipedia.org/wiki/Memoization, which, according to wikipedia comes from "memorandum"